### DISCUSSION TITLE
Not able to resolve target using tycho

### DISCUSSION THREAD

**Discussion by pramakan-tibco** [eclipse-tycho/tycho#3512] [Q&A] [answered]
Hi experts,
I am new to the world of tycho build, I tried creating a maven project as parent project and create one more maven project with target into in I trying to do a mvn clean install on the parent project build is getting successful but the target P2 is not getting resolved as nothing is getting dowloaded from repository. Please find the attached project. Kindly let me know what am I doing wrong. I have followed the demo project when importing them they are running fine.
[sampleProject.zip](https://github.com/eclipse-tycho/tycho/files/14300621/sampleProject.zip)

**Comment by laeubi** [comment]
1. you are using very old Tycho version
2. I don't see a parent project is used here that makes the target and the consumer being build together.

**Reply by pramakan-tibco** [reply]
Thanks.

**Comment by pramakan-tibco** [comment, marked as answer]
Updating version to tycho and making target and consumer project build together works for me.
